solidus_i18n.gemspec

```diff
@@ -40,4 +40,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop', '>= 0.24.1'
   s.add_development_dependency 'rspec-rails', '~> 3.1'
   s.add_development_dependency 'simplecov', '~> 0.9'
+  s.add_development_dependency 'i18n-tasks', '~> 0.9.15'
```

Comment:
I'm sure we can remove the lock to the patch version and only lock to the minor version instead.